tokio/src/task/spawn.rs

```diff
@@ -129,7 +130,7 @@ cfg_rt! {
         T::Output: Send + 'static,
     {
         let spawn_handle = runtime::context::spawn_handle()
-        .expect("must be called from the context of Tokio runtime configured with either `basic_scheduler` or `threaded_scheduler`");
+        .unwrap_or_else(|| panic!(context_missing_error(&[&"basic_scheduler", &"threaded_scheduler"])));
```

Comment:
This is not a feature flag, but a setting on the runtime. I also don't think it can happen anymore, since we removed the shell runtime with the release Tokio 1.0.0

Reply:
@Darksonn Thanks for making me aware of that. Shall I just revert this change?

Reply:
Ah, I misread. I thought it was the `Runtime::spawn` method, but it's the free-standing `spawn` function. It _can_ still fail, but the part about the basic/threaded scheduler thing should just be removed.

Reply:
OK